src/shared/components/oncoprint/ResultsViewOncoprint.tsx

```diff
@@ -123,6 +130,8 @@ export default class ResultsViewOncoprint extends React.Component<IResultsViewOn
     private heatmapGeneInputValueUpdater:IReactionDisposer;
 
     public selectedClinicalAttributeIds = observable.shallowMap<boolean>();
+    public genesetHeatmapTrackExpansionGenes =
```

Comment:
can you make more explicit in the name what the map key represents? is it the track id?

Reply:
Basically yes, it's the track key (not specific to the track group). I've gotten used to clarifying these things in the type, but sadly `ObserveableMap` doesn't allow annotating the keys with more type information than ‘any string of characters will do’. Will change the name to reflect it.